HW3_spark_smythe.py

```diff
@@ -1,156 +0,0 @@
-"""

```

Comment:
It's showing your py files as being deleted?

Reply:
Ill take a look if i should have submitted the revert version.